ISSUES.txt

```diff
@@ -1,365 +0,0 @@
-
```

Comment:
Personally I think the best way forward is to open one "Old ISSUES.txt" github issue and then start discussing what's already done, for what we want to open a new issue, and what to drop.

I've only taken a very quick look, but I'd imagine some points there would still be valid and interesting.